io.github.Tahoma2D.yaml

```diff
@@ -0,0 +1,180 @@
+app-id: io.github.Tahoma2D
+branch: stable
+runtime: org.kde.Platform
+runtime-version: 5.10
```

Comment:
I don't even think we ever had that version. `5.15-21.08` is the only Qt5 based runtime that is supported at the time.